package.json

```diff
@@ -37,31 +37,22 @@
   },
   "homepage": "https://github.com/smartdevicelink/sdl_javascript_suite#readme",
   "devDependencies": {
```

Comment:
Trying to run the unit tests with `npm run test` after clearing and reinstalling the node_modules fails and returns the error `Error: Cannot find module 'ws'`.

Reply:
Ah. I was trying for so long to figure out why we needed the 'ws' module in the package.json. I'll revert the removal of that package.